### ISSUE TITLE
False positive attribute separator in release v2.0.1

### ISSUE THREAD

**Issue by gergelykovacs** [benedfit/jade-lint#44]
Hi!

The false positive attribute separator detection by examples.

Relevant config line in `.jadelintrc`

```json
{
    ...
    "validateAttributeSeparator": ", ",
    ...
}
```

**Invalid attribute separator found**
```jade
a(target=model.target, href=model.link, title=model.label, class!=attributes.class)= model.label
a(href='tel:#{model.link}', class!=attributes.class)= model.label
a(href='<%= templateLink %>', title=model.label, class='template-clazz')= model.label
```

**Comment by gergelykovacs** [comment]
Thanks a lot! 

Now v2.0.2 seems good.
